resources/js/hiroapp/components/recruiter/CreatePosition.jsx

```diff
@@ -0,0 +1,123 @@
+import axios from "axios";
+import React, { useEffect, useState } from "react";
+import { useNavigate } from "react-router-dom";
+
+const CreatePosition = () => {
+	const navigate = useNavigate;
+
+	const [positions, setPositions] = useState(null);
+	const [values, setValues] = useState({
+		name: "",
+		description: "",
+		start_date: "",
+		end_date: "",
+		pay_grade: "",
+	});
+
+	const fetchPositions = async () => {
+		try {
+			const response = await axios.get("/api/positions/all");
+			setPositions(response.data);
+		} catch (error) {
+			console.log(error.response.data);
+		}
+	};
+
+	const handleChange = (ev) => {
+		setValues((previous_values) => {
+			return {
+				...previous_values,
+				[ev.target.name]: ev.target.value,
+			};
+		});
+	};
+
+	const handleSubmit = async (ev) => {
+		ev.preventDefault();
+		try {
+			const response = await axios.post("/api/positions", values);
+			navigate("/");
+		} catch (error) {
+			console.log(error);
+		}
+	};
+
+	useEffect(() => {
+		fetchPositions();
+	}, []);
+
+	return (
+		<>
+			<h2>Create new position</h2>
+
+			<form action="/" method="post" onSubmit={handleSubmit}>
+				<br />
+				<label>
```

Comment:
I think label is not covering the elements inside them. I checked online and I see that label is only one line element.

Reply:
https://react.dev/reference/react-dom/components/input#providing-a-label-for-an-input
https://react.dev/reference/react-dom/components/select